app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -70,11 +71,13 @@ class AboutActivity : BaseNavigationActivity() {
 
     private fun _openUrl(url: String) {
         if (URLUtil.isValidUrl(url)) {
-            val openURL = Intent(Intent.ACTION_VIEW)
-            openURL.data = Uri.parse(url)
-            if (openURL.resolveActivity(packageManager) != null) {
-                startActivity(openURL)
-            } else {
+            try {
+                val intent = Intent(Intent.ACTION_VIEW).apply {
+                    data = Uri.parse(url)
+                    flags = Intent.FLAG_ACTIVITY_NEW_TASK
+                }
+                startActivity(intent)
+            } catch (e: ActivityNotFoundException) {
```

Comment:
Check:
- [ ] This is the only possible exception `startActivity()` may throw.

If not, you might want to catch all exceptions by a general type here.
Same for opening map in RouteActivity

Reply:
Yes, it is.